kategory-effects-test/gradle.properties

```diff
@@ -0,0 +1,4 @@
+# Maven publishing configuration
+POM_NAME=Kategory-Effects-Test
+POM_ARTIFACT_ID=kategory-effects-test
```

Comment:
we could remove kategory from artifact because we already use it in group id i like `io.kategory:effects-test` instead of  `io.kategory:kategory-effects-test`

Reply:
All other packages, including the processor, use the kategory- prefix. I've kept it for consistency.